java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/menu/domain/Menu.java

```diff
@@ -13,7 +13,7 @@
 @Data
 @AllArgsConstructor
 @Builder
-@Document(collection = "Menu")
+@Container(containerName = "Menu")
```

Comment:
Why do you switch from Document to Container?

Reply:
In the new Azure API Document doesn't exist, it took me a while but they use the Container annotation now for specifying DB collections.

Reply:
A previous iteration of CosmosDB was called DocumentDB, the naming conventions changed when they changed the name.

Reply:
Cool. So  "Container" will be!